unfoldingWord/create-app
# Implemented tN - tA article navigation

## Test instructions

- Open https://deploy-preview-22--create-app.netlify.app/
- Verify that navigating through notes that have a support reference renders a tA article in the tA card



This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword/create-app/22)
